tests/lldp/test_lldp.py

```diff
@@ -83,6 +83,8 @@ def test_lldp_neighbor(duthosts, enum_rand_one_per_hwsku_frontend_hostname, loca
         nei_lldp_facts = localhost.lldp_facts(
             host=hostip, version='v2c', community=eos['snmp_rocommunity'])['ansible_facts']
         neighbor_interface = v['port']['ifname']
+        logger.info("lldp facts for interface {}:{}".format(neighbor_interface,
+                                                            nei_lldp_facts['ansible_lldp_facts'][neighbor_interface]))
```

Comment:
Is this log only used for this issue debug? or we need it permanently? thanks.
If for debugging, suggest to add it in internal branch and dump more information.